CONTRIBUTING.md

```diff
@@ -1,5 +1,47 @@
-# Contribute
+# Contribute to the documentation
+We really appreciate your support! Please have a look at the [issues page on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues) and check if anybody already raised an issue with the same topic.
 
-Thanks for your support!
-Please open an issue on github. More to come soon...
-~~Please open a pull request on GitHub!~~
+If this is the case, please subscribe to the issue and explain your proposal.
+
+If that's not the case, please open a [new issue](https://github.com/jamulussoftware/jamuluswebsite/issues/new/choose), select which type of issue it is and describe your issue.
+
+If you want to open a Pull Request (after having described and discussed your proposal), have a look at the [Contribution page on the website](https://jamulus.io/wiki/Contribution). Keep in mind that your changes (if they are not just typos) might have to wait until everything is translated. We will bundle them (if everything is allright) with other changes and upload multiple changes in one bulk to jamulus.io.
+
+## I don't know GitHub. What do I need to know?
+If you aren't comfortable with Pull Requests, commits, etc. just open an issue. We will then care about your changes. If you want to do the changes on your own and open a Pull Request, follow this instructions:
+1. Fork this repo (there's a fork button on the top right of the page)
+2. Descide which branch you have to commit your changes too (rule of thumb: everything which might need translation should go to the `changes` branch, typos should go to the `release` branch more information can be found later in this file). Create a new branch from the up to date branch you chose
+3. Make your changes
+4. If you did the changes offline, push the new branch into your repository
+5. Submit your Pull Request, give a short description of your changes and link the issue you opened
+6. Wait for a review and a merge.
+
+Hint (for git): **Do not merge your work** into your local `changes` branch, only keep it in sync by pooling from the remote Jamulus Website repository. Create a new branch for your proposed changes.
```

Comment:
"Website" shouldn't be capitalised unless there is a good reason for that.
(Hint: it's not capitalised on another occurrence above)